draft-nuttycom-funding-allocation.rst

```diff
@@ -0,0 +1,460 @@
+upper value, to ensure that funding remains available for years to come.
+
+Alternative 2
+-------------
+
+Proposed by Jason McGee
+
+* 12% of the block subsidy is to be distributed to the lockbox.
+* 8% of the block subsidy is to be distributed to the Financial Privacy
+  Foundation (FPF), for the express use of the Zcash Community Grants Committee
+  (ZCG) to fund independent teams in the Zcash ecosystem.
+
+As of block height 2726400, and continuing for one year, the complete set of
+funding streams will be::
+
+  ================= =========== ============= ============== ============
+        Stream       Numerator   Denominator   Start height   End height
+  ================= =========== ============= ============== ============
+  ``FS_DEFERRED``       12           100          2726400      3146400
+  ``FS_FPF_ZCG``         8           100          2726400      3146400
+  ================= =========== ============= ============== ============
+
+Motivations for Alternative 2
+'''''''''''''''''''''''''''''
+
+* **Limited Runway**: ZCG does not have the financial runway that ECC/BP and ZF
+  have. As such, allocating ongoing funding to ZCG will help ensure the Zcash
+  ecosystem has an active grants program.
+
+* **Promoting Decentralization**: Allocating a portion of the Dev Fund to Zcash
+  Community Grants ensures small teams continue to receive funding to
+  contribute to Zcash. Allowing the Dev Fund to expire, or putting 100% into a
+  lockbox, would disproportionally impact grant recipients. This hybrid
+  approach promotes decentralization and the growth of independent development
+  teams.
+
+* **Mitigating Regulatory Risks**: By minimizing direct funding of US-based
+  organizations, the lockbox helps to reduce potential regulatory scrutiny and
+  legal risks.
+
+Alternative 3
+-------------
+
+Proposed by Kris Nuttycombe
+
+* 20% of the block subsidy is to be distributed to the lockbox.
+
+As of block height 2726400, and continuing for two years, the complete set of
+funding streams will be::
+
+  ================= =========== ============= ============== ============
+        Stream       Numerator   Denominator   Start height   End height
+  ================= =========== ============= ============== ============
+  ``FS_DEFERRED``       20           100          2726400      3566400
+  ================= =========== ============= ============== ============
+
+Motivations for Alternative 3
+'''''''''''''''''''''''''''''
+
+This alternative is presented as the simplest allocation of block rewards
+to a lockbox for future disbursement that is consistent with results of
+community polling.
+
+Alternative 4
+-------------
+
+Proposed by NoamChom (Zcash forum)
+
+* 17% of the block subsidy is to be distributed to the lockbox.
+* 8% of the block subsidy is to be distributed to the Financial Privacy
+  Foundation (FPF), for the express use of the Zcash Community Grants Committee
+  (ZCG) to fund independent teams in the Zcash ecosystem.
```

Comment:
Is this incorporating #857 (which allocates 15%), or #868 (which allocates 20%), or is this a new third option (which allocates 25%)?

Reply:
This is incorporating https://github.com/noamchom67/MoU/blob/main/Masters_of_the_universe_ZIP_proposal.md

Reply:
Aha, I worked out why I thought #868 only allocated 20%: the abstract allocates 25%, while the specification allocates 20%. So currently it is unclear whether this ZIP is correctly incorporating #868.

Reply:
I believe the 20% is a copy/paste error, based on what @noamchom67 said in his post to the forum.

Reply:
correct, existing 20% was missed on my behalf.
Intention is 25% split as 17 and 8. posted from mobile.